$OpenBSD: patch-src_main_c,v 1.1 2014/01/28 17:03:59 jturner Exp $

Relax the SQLite version check for this release. This patch should be
removed by next release as Fossil will require at least 3.8.2.

--- src/main.c.orig	Mon Jan 27 15:46:41 2014
+++ src/main.c	Mon Jan 27 15:49:19 2014
@@ -570,8 +570,8 @@ int main(int argc, char **argv)
   const char *zCmdName = "unknown";
   int idx;
   int rc;
-  if( sqlite3_libversion_number()<3008002 ){
-    fossil_fatal("Unsuitable SQLite version %s, must be at least 3.8.2",
+  if( sqlite3_libversion_number()<3008000 ){
+    fossil_fatal("Unsuitable SQLite version %s, must be at least 3.8.0",
                  sqlite3_libversion());
   }
   sqlite3_config(SQLITE_CONFIG_SINGLETHREAD);
